src/Module.js

```diff
@@ -51,9 +51,12 @@ class Module extends Dispatcher {
     this._name = name;
     this.loaded = false
     app.on('stop', this.removeAllListeners.bind(this));
-    this._app.on('load.before', () => {
+    app.on('load.before', () => {
       this.loaded = true
     })
+    this._requestedEvents = {}
+    this._registeredEvents = {}
+    app.after('launch', this._checkMissingEvents.bind(this))
```

Comment:
So warnings only show up after launch? Why not have them show up as they happen?  Would be nice to know if I was calling an event before it had been registered, even if technically valid.